spaceship/lib/spaceship/portal/app.rb

```diff
@@ -15,6 +27,10 @@ class App < PortalBase
       # @return (String) the supported platform of this app
       # @example
       #   "ios"
+      #   "mac"
```

Comment:
The point being, You can always pass, watchOS, tvOS, iOS, macOS, futureOS. Even though some  of them are "sub -platforms" on the web. And who knows, one day they could suddenly no longer be sub-platforms and going this approach would mean that only spaceship would need to be updated nothing else.